.azure-pipelines.yml

```diff
@@ -97,12 +97,13 @@ jobs:
     displayName: 'Run tests (Chrome Dev)'
   - script: no_proxy='*' ./wpt run --yes --no-manifest-update --manifest MANIFEST.json --metadata infrastructure/metadata/ --log-tbpl - --log-tbpl-level info --channel nightly firefox infrastructure/
     displayName: 'Run tests (Firefox Nightly)'
-  - script: no_proxy='*' ./wpt run --yes --no-manifest-update --manifest MANIFEST.json --metadata infrastructure/metadata/ --log-tbpl - --log-tbpl-level info --channel preview safari infrastructure/
+  - script: no_proxy='*' ./wpt run --yes --no-manifest-update --manifest MANIFEST.json --metadata infrastructure/metadata/ --log-tbpl - --log-tbpl-level info --webdriver-arg=--diagnose --channel preview safari infrastructure/
     displayName: 'Run tests (Safari Technology Preview)'
   - task: PublishBuildArtifacts@1
-    displayName: 'Publish results'
```

Comment:
I'm replacing this task rather than adding a new one because it hasn't been updating any artifacts since https://github.com/web-platform-tests/wpt/commit/7dd31d2b2f9ebca522b50af958c18208d08683ae, and could have been removed then. @Hexcles FYI.